src/scripts/index.js

```diff
@@ -16,20 +16,29 @@ function init() {
     var box = document.createElement('div');
     box.id = 'stravaKudos';
 
-    var btn = document.createElement('span');
-    btn.innerHTML = 'Yup!';
-    btn.onclick = function() {
+    var yup = document.createElement('span');
+    yup.innerHTML = 'Yup!';
+    yup.onclick = function() {
       box.parentNode.removeChild(box);
 
       for (var i = 0; i < kudosBtns.length; i++) {
         kudosBtns[i].click();
       }
     }
+    
+    var cancel = document.createElement('span');
+    cancel.innerHTML = 'Nah.';
+    cancel.onclick = function() {
+      box.parentNode.removeChild(box);
+    }
+    
     box.innerHTML = 'There are ' + kudosBtns.length + ' activities that you havent Kudos\'d, would you like to?';
-    box.appendChild(btn);
+    box.appendChild(yup);
+    box.appendChild(cancel);
 
-    document.body.appendChild(box);;
+    document.body.appendChild(box);
+    clearInterval(initInterval);
```

Comment:
the `setInterval` is there to account for new Kudos buttons that appear when scrolling. Ideally we only do a scan when we scroll (throttled so we don't overload the browser with events). In the meantime, we could `clearInterval` upon clicking the 'Nah' button only.